team/_posts/2017-04-06-bottenhorn-katherine.md

```diff
@@ -22,4 +22,8 @@ site:
 twitter: 62442katieb
 ---
 
+<<<<<<< HEAD:team/_posts/2017-01-06-katherine-bottenhorn.md
```

Comment:
The <<<<<<, =====, and >>>>>> sections are indicating merge conflicts that were saved rather than resolved. Can you remove the old version of the paragraph? The same issue occurs in `news/_posts/2017-09-30-poudel-psychiatryletter.md` as well.

Reply:
yeah, my b. i only saw those after I fetched the upstream

Reply:
should be good now, I think...